<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropPageQML</class>
 <widget class="QWidget" name="PropPageQML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control panel QML</string>
  </property>
  <widget class="QLabel" name="labelPathToQml">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path to QML:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQmlFile">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonSetQmlFile">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="qlocalcontrolqmlplugin.qrc">
     <normaloff>:/Resources/standardbutton-open-32.png</normaloff>:/Resources/standardbutton-open-32.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelHotKeys">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>50</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hot keys:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGP10">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>81</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GP10</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg3">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>81</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+4</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTA0">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TA0</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg5">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>109</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+6</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAR15">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>109</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AR15</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>109</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+5</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCP1">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CP1</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg2">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSPEC">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>109</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>СПЕЦ</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg6">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>109</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+7</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTARG">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ЦЕЛЬ</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg7">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+8</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPRG2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PRG2</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Ctrl+2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPRG1">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>80</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PRG1</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPrg0</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrg0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>C...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qlocalcontrolqmlplugin.qrc"/>
 </resources>
 <connections/>
</ui>
